### PR TITLE
Db 1772 display certified by

### PR DESCRIPTION
**Pull request by alburde1**
Display who certified submissions were most recently certified by instead of the delete column in the submission dashboard. Delete column for published/updated submissions in the recent activity table now displays "N/A" instead of a delete button.

- [x] Merged after or concurrently with https://github.com/fedspendingtransparency/data-act-broker-backend/pull/626
- [x] Reviewed by front end

### CODE REVIEW

**Comment by kkuratsu** [comment]
You might want to merge dev on to this working branch to resolve the merge conflicts?

**Comment by alburde1** [comment]
Working on the back end of this first for resolving merge conflicts

File: src/js/components/landing/recentActivity/RecentActivityTable.jsx [hunk 1]
```diff
@@ -151,10 +151,16 @@ export default class RecentActivityTable extends React.Component {
 				reportingDateString,
 				userName,
 				item.last_modified,
-				<Status.SubmissionStatus status={item.rowStatus} certified={item.publish_status !== 'unpublished'} />,
-				<DeleteLink submissionId={item.submission_id} index={index} warning={this.deleteWarning.bind(this)} confirm={deleteConfirm} reload={this.reload.bind(this)} item={item} account={this.state.account}/>
+				<Status.SubmissionStatus status={item.rowStatus} certified={item.publish_status !== 'unpublished'} />
 			];
 
+			if(item.publish_status === "unpublished") {
```
Comment (kkuratsu) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: We only allow deletion on `unpublished` only and not `updated`, right?

Comment (alburde1) [inline src/js/components/landing/recentActivity/RecentActivityTable.jsx]: Yes, only unpublished submissions can be deleted. If they've ever been certified they can no longer be deleted.

**Pull request merged**